src/libclipper/test/serialization_test.cpp

```diff
@@ -350,37 +356,27 @@ namespace {
   }
 
   TEST(SerializationTests, ByteVectorsHashCorrectly) {
```

Comment:
These aren't serialization tests.

Can you rename the test case for the tests you added to "InputHashTests" and rename the existing tests to "InputSerializationTests". Then let's rename this file to `input_test.cpp`.